### ISSUE TITLE
Flickering fix issued marker + drawline in combination

### ISSUE THREAD

**Issue by C0kkie** [altmp/altv-issues#982]
**Client/server version**
`rc/4.0-rc1`

**Current behavior**
You cant use drawline in combination with marker native

**Expected behavior**
Both should work

**Steps to reproduce**
```
const player = alt.Player.local;
alt.everyTick(() => {
    let pos = player.pos;

    native.drawRect(0, 0, 0, 0, 0, 0, 0, 0, false);

    native.drawBox(pos.x, pos.y, pos.z - 0.5, pos.x + 5, pos.y + 5, pos.z - 0.5, 0, 255, 0, 255);
    native.drawLine(pos.x, pos.y, pos.z, pos.x + 5, pos.y + 5, pos.z, 255, 0, 0, 255);

    //with this line the line and box are not visible
    //native.drawMarker(1, pos.x, pos.y, pos.z, 0, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 255, 255, false, false, 2, false, undefined, undefined, false);
});
```

**Comment by pixlcrashr** [comment]
@C0kkie Idk if it is related to this, but using the `drawMarker` native may result in flickering dialogs. It may be because of multiple `drawMarker` calls. I'll have a look into it.

**Comment by C0kkie** [comment]
that code i posted is enough to reproduce, but there is no eta to fix

**Comment by pixlcrashr** [comment]
> that code i posted is enough to reproduce, but there is no eta to fix

I meant that using `drawMarker` results in flickering CEF dialogs. I didn't saw an issue for that specifically.

**Comment by pixlcrashr** [comment]
@C0kkie A final note: When looking at a `drawMarker`-Marker, CEF pages start to flicker - including the alt:V UI.

Current solution is to disable markers completely.

**Comment by ThomasMarangoni** [comment]
@pixlcrashr this is a known GTA Bug, if no GTA Hud is shown and a marker is on the screen. You can fix this with the native draw rect of sizes 0x0 in everyTick. But it isn't related to this bug.

**Comment by pixlcrashr** [comment]
> @pixlcrashr this is a known GTA Bug, if no GTA Hud is shown and a marker is on the screen. You can fix this with the native draw rect of sizes 0x0 in everyTick.

First of all, thanks for the answer. It works, though.

I am curious why that is.

**Comment by C0kkie** [comment]
We cant repro this on rc/11.0-rc21 anymore. If someone has this bug, please write us

**Comment by C0kkie** [comment]
Will be fixed in dev/15.0-dev50
